pkgs/by-name/su/surfer/package.nix

```diff
@@ -0,0 +1,71 @@
+{
+  lib,
+  fetchFromGitLab,
+  rustPlatform,
+  pkg-config,
+  openssl,
+  wayland,
+  autoPatchelfHook,
+  libxkbcommon,
+  libGL,
+  libX11,
+  libXcursor,
+  libXi,
+  stdenv,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "surfer";
+  version = "0.2.0";
+
+  src = fetchFromGitLab {
+    owner = "surfer-project";
+    repo = "surfer";
+    rev = "v${version}";
+    hash = "sha256-C5jyWLs7fdEn2oW5BORZYazQwjXNxf8ketYFwlVkHpA";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    autoPatchelfHook
+  ];
+
+  buildInputs = [
+    openssl
+    wayland
+    libxkbcommon
+    libGL
+    stdenv.cc.cc.libgcc
+    libX11
+    libXcursor
+    libXi
+  ];
+
+  runtimeDependencies = [
+    openssl
+    wayland
+    libxkbcommon
+    libGL
+  ];
```

Comment:
Very sorry for the extra trouble, but I misplaced the last comment about extra libraries. They were meant for `runtimeDependencies`. Like here: https://github.com/dramforever/nixpkgs/commit/72d029c70275e0ed2b41555787ad0dd7c4b28fca

Again sorry for the trouble 🙇

Reply:
Nah, its my bad for not testing it on X11. I should probably create hydra like VM cluster, each having different wm and arch and run my CIs there.

Reply:
Well, the important thing is we got it working. Thanks!